scripts/zmdc.pl.in

```diff
@@ -69,7 +69,12 @@ use IO::Handle;
 use autouse 'Pod::Usage'=>qw(pod2usage);
 #use Data::Dumper;
 
-use constant SOCK_FILE => $Config{ZM_PATH_SOCKS}.'/zmdc'.$Config{ZM_SERVER_ID}.'.sock';
+if ( ! defined('ZM_SERVER_ID') ) {
```

Comment:
Won't the expression Perl```! defined('ZM_SERVER_ID')`` always return false because ZM_SERVER_ID represents a string? Perl is pretty good at truth.... try perl```if ( $Config{ZM_PATH_SOCKS} )``` and then reverse the logic under the block.